test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/jbpm/TimerIntegrationTest.java

```diff
@@ -46,7 +47,7 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
-@Category({OperatorNotSupported.class, JBPMOnly.class}) // Because TimerServiceDataStoreRefreshInterval is not supported yet, Operator is skipped as scenario use different startup strategy than template.
+@Category({OperatorNotSupported.class, JBPMOnly.class, MonitoringK8sFs.class}) // Because TimerServiceDataStoreRefreshInterval is not supported yet, Operator is skipped as scenario use different startup strategy than template.
```

Comment:
Is this correct? This integration test will be executed in Templates only and the MonitoringK8sFs is only for Operator.